extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -55,10 +55,20 @@
   top: 0 !important;
   height: 50px !important;
   background: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0));
-  transition: opacity 1s;
   z-index: 2;
 }
 
+.i-amphtml-lbv-controls {
+  opacity: 0;
+  animation: fadeIn ease-in 0.4s 1 forwards;
+}
+
+.i-amphtml-lbv-controls.fade-out
+ {
+  opacity: 0;
```

Comment:
the opacity should not be needed here right? (or should be 1 to match the animation)